### PR TITLE
added encoding code

### PR DESCRIPTION
**Pull request by akanksha-thaware**
Screenshot_ps

**Results** :

Screenshot_2

Screenshot_1

### CODE REVIEW

**Comment by daraqshan-zemoso** [comment]
add the screenshot of the component if developing something or the question if it is an assignment

**Comment by akanksha-thaware** [comment]
> add the screenshot of the component if developing something or the question if it is an assignment

I have attached the question.

File: encode.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Data Encryption System: Rahul wants to send encoded messages to his friend Ram. 
+// Develop an algorithm to encode all the digits, special characters, lower and upper case alphabets.
+
+
+#include<iostream>
+#include<string>
+
+using namespace std;
+int main(){
+    int i;
```
Comment (daraqshan-zemoso) [inline encode.cpp:10]: move the code to a method and call the method in main

**Review by daraqshan-zemoso — CHANGES_REQUESTED** [review]

File: encode.cpp (hunk 1 above)
Comment (akanksha-thaware) [reply encode.cpp:11]: I have made the changes. please review :)

**Review by daraqshan-zemoso — APPROVED** [review]

**Pull request merged**